packages/rollup-plugin-html/src/assets/utils.ts

```diff
@@ -143,3 +144,11 @@ export function getSourcePaths(node: Element) {
 export function findAssets(document: Document) {
   return findElements(document, isAsset);
 }
+
+// picomatch follows glob spec and requires "./" to be removed for the matcher to work
+// it is safe, because with or without it resolves to the same file
+// read more: https://github.com/micromatch/picomatch/issues/77
+const removeLeadingSlash = (str: string) => (str.startsWith('./') ? str.slice(2) : str);
```

Comment:
can't we just `path.normalize` it?

Reply:
I wanted to keep modifications to the original paths used in source files to a minimum